tests/lib/custom_lib/src/main.c

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2021 Legrand North America, LLC.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*
+ * @file test custom_lib library
+ *
+ * This suite verifies that the methods provided with the custom_lib
+ * library works correctly.
+ */
+
+#include <zephyr.h>
+#include <ztest.h>
+#include <custom_lib.h>
+
+static void test_get_value(void)
+{
+	int values[] = { INT_MIN, INT_MIN + 1, -1, 0,
+			 1, INT_MAX - 1, INT_MAX };
+
+	/* Verify standard behavior */
+	for (int i = 0; i < ARRAY_SIZE(values); ++i) {
+		int value = values[i];
+
+		if (value != 0) {
+			zassert_equal(custom_lib_get_value(value), value,
+				"get_value failed non-zero input");
+		}
+	}
+
+	/* Verify override behavior */
+	zassert_equal(custom_lib_get_value(0), CONFIG_CUSTOM_LIB_GET_VALUE_DEFAULT,
+		"get_value failed zero input");
+
+}
+
+void test_main(void)
+{
+	ztest_test_suite(lib_custom_lib_test,
+			 ztest_unit_test(test_get_value)
+			 );
+
+	ztest_run_test_suite(lib_custom_lib_test);
```

Comment:
unit tests should be run on CI

Reply:
Accepted.  A `testcase.yml` has been added, and run/verified by `twister` once it was told where the testcase root was.

Reply:
I can't see the test being run on the github workflow, can you add a new step on the github ci that runs twister?

Reply:
@gmarull The .github ci has now been extended to run twister on all tests in this repository against the qemu_cortex_m0 platform.